content/200-concepts/200-database-connectors/03-postgresql.mdx

```diff
@@ -180,31 +180,31 @@ When introspecting a PostgreSQL database, the database types are mapped to Prism
 | `oid`                                       | `Int`         |    ✔️     | `@db.Oid`                                |
 | `uuid`                                      | `String`      |    ✔️     | `@db.Uuid`                               |
 | `json`                                      | `Json`        |    ✔️     | `@db.Json`                               |
-| `jsonb`                                     | `Json`        |    ✔️     | `@db.JsonB`\*                            | \*Default mapping for `Json` - no type attribute added to schema.     |
-| `bytea`                                     | `Bytes`       |    ✔️     | `@db.ByteA` \*                           |                                                                       | \*Default mapping for `Bytes` - no type attribute added to schema. |
+| `jsonb`                                     | `Json`        |    ✔️     | `@db.JsonB`\*                            | \*Default mapping for `Json` - no type attribute added to schema.                                                                                                               |
+| `bytea`                                     | `Bytes`       |    ✔️     | `@db.ByteA` \*                           |                                                                                                                                                                                 | \*Default mapping for `Bytes` - no type attribute added to schema. |
```

Comment:
Looks like this description is on 6th column, but should be on 5th, and therefore is not visible now now http://localhost:8000/concepts/database-connectors/postgresql/#native-type-mappings

Looks like that: 

<img width="606" alt="image" src="https://user-images.githubusercontent.com/4734297/125613358-d7511e91-f34a-46c5-b62d-c63ec746eedb.png">

Prettier normalized table and add 1 extra column for each of row, and now looks like that:

<img width="651" alt="image" src="https://user-images.githubusercontent.com/4734297/125613641-65c7bd2e-5b5f-4473-8d8b-b9845d965b9b.png">